src/main/java/com/scottlogic/practice/mealscheduler/Models/Meal.java

```diff
@@ -1,25 +1,39 @@
 package com.scottlogic.practice.mealscheduler.Models;
 
-import lombok.Data;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
 
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
 import java.time.LocalDateTime;
-import java.util.UUID;
 
-@Data
+/*not using @Data class as IntelliJ says it is bad to use it with @Entity*/
+@Entity
+@NoArgsConstructor
+@AllArgsConstructor
 public class Meal {
-    private UUID Id;
+    public static Meal copyOf(Meal meal) {
+        return new Meal(meal.getId(), meal.getName(), meal.getTime(), meal.getUser());
+    }
```

Comment:
Is this better practice than having a constructor that takes a Meal as an argument?

Reply:
More or less same thing.... somehow now those static method based, 'builders' are in fashion. Was wondering whether adding a new constructor 'directly' where there are indirect (annotation based) ones, would not look confusing/messy.